spell/Frog God Games; Book of Lost Spells.json

```diff
@@ -3870,7 +3870,7 @@
 				]
 			},
 			"entries": [
-				"You conjure a blade of light in your hand. Treat this blade as a longsword +1, +2 vs. evil creatures. The blade emits bright light in a 20-foot radius."
+				"You conjure a blade of light in your hand. Treat this blade as a {@item longsword +1|+1 longsword|dmg}, +2 vs. evil creatures. The blade emits bright light in a 20-foot radius."
```

Comment:
it's `{@<tag> <5etools name>|<source>|<display name>}`, so this is broken. Should be `{@item +1 longsword}` if you want it to show up as `+1 longsword`

Reply:
Thanks for that, I'll re-do all these with the correct format!